### DISCUSSION TITLE
Add fees for different payment methods

### DISCUSSION THREAD

**Discussion by OddDevelopment** [Paymenter/Paymenter#450] [Ideas]
Add something like the taxes section that lets you add on an extra % fee (For example stripe takes big fees so it would be nice to charge extra to make up for it)

**Comment by GomorrhaDev** [comment]
However, if you live in the EU, please note that it is illegal to charge fees on a credit or debit card payment, but you can surcharge Stripes fees. (And it is only legal to surcharge the amount you pay).
